src/statsmodel.jl

```diff
@@ -131,26 +131,26 @@ StatsBase.adjr2(mm::TableRegressionModel) = adjr2(mm.model)
 StatsBase.r2(mm::TableRegressionModel, variant::Symbol) = r2(mm.model, variant)
 StatsBase.adjr2(mm::TableRegressionModel, variant::Symbol) = adjr2(mm.model, variant)
 
-function _return_predictions(yp::AbstractVector, nonmissings, len)
+function _return_predictions(T, yp::AbstractVector, nonmissings, len)
     out = missings(eltype(yp), len)
     out[nonmissings] = yp
     out
```

Comment:
Note that in this case, we return a mere vector (not a data frame) even if the input is a data frame. I guess that's the most logical output.

Reply:
I think that's what the consensus was right?  I remember their being some extensive discussion about this...

Reply:
Ah yes that was https://github.com/JuliaStats/StatsModels.jl/pull/81.